src/features/classification/applied-by-ai-classification-reason/AppliedByAiClassificationReason.scss

```diff
@@ -0,0 +1,31 @@
+@import '~@box/blueprint-web-assets/dist/tokens/tokens';
+
+$reason-font-size: 13px; // Overriding Blueprint to match current sidebar styles
+
+.AppliedByAiClassificationReason {
+    .AppliedByAiClassificationReason-headerText {
+        font-size: $reason-font-size;
+    }
+
+    .AppliedByAiClassificationReason-answer {
+        font-size: $reason-font-size;
+        line-height: $body-default-line-height;
+    }
+
+    .AppliedByAiClassificationReason-references {
+        > * > * {
```

Comment:
is the reason you need `> * > *` due to specificity? if so, maybe try repeating the classnames like:

```
.AppliedByAiClassificationReason-references.AppliedByAiClassificationReason-references {
```

otherwise it's sort of fragile based on the DOM structure

Reply:
ah it's more so because we want to override the styling within the Blueprint elements so the font size is in parity. irrc it doesn't take a classname atm and it's a sub element, so from offline discussion we've decided to go this route for now: similarly, https://git.dev.box.net/Box/EndUserApp/pull/14016/files#diff-059537c7a248b3eb2e81c985f5f42fdefeb8d1788a2a1519bce72c87149c04a1R27-R30